aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -540,7 +551,7 @@ void gemm<at::BFloat16>(CUDABLAS_GEMM_ARGTYPES(at::BFloat16)) {
 }
 #endif
 
-#if !defined(USE_ROCM)
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
```

Comment:
@zstreet87 This change seems to be deviating from convention by ifdefing at the function definition level itself, instead of at the point where rocblas/cublas Gemm apis are called. Why?